samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -149,232 +210,116 @@ static void state_binding(u8_t lightness, u8_t temperature)
 				light_lightness_srv_user_data.actual =
 					light_lightness_srv_user_data.def;
 			}
-
-			tmp = ((float)
-			       light_lightness_srv_user_data.actual / 65535);
-			light_lightness_srv_user_data.linear =
-				(u16_t) (65535 * tmp * tmp);
-
-			light_lightness_srv_user_data.last =
-				light_lightness_srv_user_data.actual;
-
-			gen_level_srv_root_user_data.level =
-				light_lightness_srv_user_data.actual - 32768;
-
-			light_ctl_srv_user_data.lightness =
-				light_lightness_srv_user_data.actual;
 		}
-
-		goto update_temp;
-
 		break;
 	case LEVEL: /* Lightness update as per Generic Level (root) state */
-		/* This is as per Mesh Model Specification 3.3.2.2.3 */
 		tmp16 = gen_level_srv_root_user_data.level + 32768;
-		if (tmp16 > 0 && tmp16 <
-		    light_lightness_srv_user_data.lightness_range_min) {
-			tmp16 =
-			light_lightness_srv_user_data.lightness_range_min;
-		} else if (tmp16 >
-			   light_lightness_srv_user_data.lightness_range_max) {
-			tmp16 =
-			light_lightness_srv_user_data.lightness_range_max;
-		}
-
-		light_lightness_srv_user_data.actual = tmp16;
-
-		tmp = ((float) light_lightness_srv_user_data.actual / 65535);
-		light_lightness_srv_user_data.linear =
-			(u16_t) (65535 * tmp * tmp);
-
-		light_lightness_srv_user_data.last =
-			light_lightness_srv_user_data.actual;
-
-		gen_level_srv_root_user_data.level =
-			light_lightness_srv_user_data.actual - 32768;
-
-		light_ctl_srv_user_data.lightness =
-			light_lightness_srv_user_data.actual;
+		constrain_lightness_actual_state(tmp16);
+		update_gen_onoff_state();
 		break;
 	case DELTA_LEVEL: /* Lightness update as per Gen. Level (root) state */
 		/* This is as per Mesh Model Specification 3.3.2.2.3 */
 		tmp16 = gen_level_srv_root_user_data.level + 32768;
 		if (tmp16 > 0 && tmp16 <
-		    light_lightness_srv_user_data.lightness_range_min) {
+			light_lightness_srv_user_data.lightness_range_min) {
```

Comment:
Looks like you've broken the alignment here and split the line in the wrong place. Split it after `&&` and align it with the ( like it used to be.

Reply:
done !!